model/src/test/kotlin/config/OrtConfigurationTest.kt

```diff
@@ -192,6 +198,30 @@ class OrtConfigurationTest : WordSpec({
                 }
             }
         }
+
+        "support environmental variables" {
+            val user = "user"
+            val password = "password"
+            val url = "url"
+            val schema = "schema"
+            val env = mapOf("ort.scanner.storages.postgresStorage.username" to user,
+                "ort.scanner.storages.postgresStorage.url" to url,
+                "ort__scanner__storages__postgresStorage__schema" to schema,
+                "ort__scanner__storages__postgresStorage__password" to password)
```

Comment:
Please format this like

            val env = mapOf(
                "ort.scanner.storages.postgresStorage.username" to user,
                "ort.scanner.storages.postgresStorage.url" to url,
                "ort__scanner__storages__postgresStorage__schema" to schema,
                "ort__scanner__storages__postgresStorage__password" to password
            )